ddtrace/internal/ci_visibility/utils.py

```diff
@@ -139,3 +139,8 @@ def take_over_logger_stream_handler(remove_ddtrace_stream_handlers=True):
     root_logger.setLevel(min(root_logger.level, ci_visibility_handler.level))
 
     log.debug("logger setup complete")
+
+
+def combine_url_path(*args: str):
+    """Combine URL path segments."""
+    return "/".join(str(segment).strip("/") for segment in args)
```

Comment:
Nice solution :ok_hand: 

The only potential problem I see is that it will remove a trailing `/` from an URL, but in our case I think it's not a problem.

Reply:
It's not a problem for our current usage but... I'm going to add a comment that it should be used with care. 😬